ohdear/resource_site.go

```diff
@@ -1,115 +1,96 @@
 package ohdear
 
 import (
-	"encoding/json"
 	"fmt"
-	"strconv"
+	"io/ioutil"
+	"log"
 
+	"github.com/articulate/ohdear-sdk/ohdear"
 	"github.com/hashicorp/terraform/helper/schema"
-	"github.com/smallnest/goreq"
 )
 
-type Site struct {
-	Url    string `json:"url"`
-	TeamId string `json:"team_id"`
-}
-
 func resourceOhdearSite() *schema.Resource {
 	return &schema.Resource{
-		Create: resourceSiteCreate,
-		Read:   resourceSiteRead,
-		Update: resourceSiteUpdate,
-		Delete: resourceSiteDelete,
-		Exists: resourceSiteExists,
+		Create: resourceOhdearSiteCreate,
+		Read:   resourceOhdearSiteRead,
+		Delete: resourceOhdearSiteDelete,
 		Schema: map[string]*schema.Schema{
+			"site_id": &schema.Schema{
```

Comment:
Is there a reason to have a `site_id` and not use the built in ID? I see it is an int but since it is computed, and not user inputed, that is OK, it just needs to be converted. Am I missing something there?

Reply:
No, I think I was just confused over the function of the `Id` member in `ResourceData`